gems/sorbet-runtime/lib/types/props/decorator.rb

```diff
@@ -302,6 +317,27 @@ def prop_validate_definition!(name, cls, rules, type)
     T::Utils::Nilable.is_union_with_nilclass(cls) || ((cls == T.untyped || cls == NilClass) && rules.key?(:default) && rules[:default].nil?)
   end
 
+  sig(:final) { params(name: Symbol).returns(T::Boolean).checked(:never) }
+  private def is_override?(name)
+    @class.method_defined?(name) && !@class.method_defined?(name, false)
+  end
+
+  sig(:final) { params(name: Symbol, rules: Rules).void.checked(:never) }
+  private def validate_overrides(name, rules)
+    override = elaborate_override(name, rules[:override])
+
+    return if rules[:without_accessors]
```

Comment:
Can we add a test that exercises this branch?

Reply:
The `without_accessors` case gets exercised implicitly in `gems/sorbet-runtime/test/types/props/struct.rb` (which is where I found that this was necessary in the first place). I can add an explicit test if you'd prefer, though.

Reply:
Interesting, I don't see any uses of `without_accessors` in `test/types/props/struct.rb` that seems to also be in an inheritance chain. So I guess maybe my question is: can you tell me more about what you found that makes this necessary?

Reply:
Actually, you're right, the case I was thinking of actually doesn't need this.

The case in question was [this one](https://github.com/sorbet/sorbet/blob/master/gems/sorbet-runtime/test/types/props/struct.rb#L8-L10) using `without_accessors` to clobber the builtin `class` accessor. In this case, it would be fine (because we aren't erroring on implicit overrides), but we'll need explicit `without_accessors` handling eventually.